packages/gatsby-theme-api-docs/src/components/resource/method/method.js

```diff
@@ -138,6 +138,7 @@ const Method = ({
 function computeMethodColor(methodName) {
   switch (methodName) {
     case 'post':
+    case 'put':
```

Comment:
Maybe we can map each case explicitly and throw an error in the `default` case.

Reply:
I don't get what you mean by mapping each case explicitly.

Reply:
Something like this:

```js
switch (methodName) {
  case 'post':
  // ...
  default:
    throw new Error(`Unknown method ${methodName}`)
}
```

Reply:
Also, I'm not sure why an error needs to be thrown here, if the method does not exist this component would not render. Can you please explain your reasoning?

Reply:
Personally, I would rather not include a default case. Only reason I did that is because of eslint. What do you think about this approach?

Reply:
The intent is to help preventing "bugs", for example here by forgetting to map a method.

If all the methods are mapped, there is no problem. If a method is not mapped though, there will be an error. Does that make sense?